src/Conversion/ONNXToKrnl/Tensor/Resize.cpp

```diff
@@ -147,65 +137,83 @@ struct ONNXResizeOpLowering : public ConversionPattern {
     outputLoops.createDefineAndIterateOp(alloc);
     rewriter.setInsertionPointToStart(outputLoops.getIterateBlock());
 
+    // Constants used in the loop body
+    Value zero = emitConstantOp(rewriter, loc, rewriter.getIntegerType(64), 0);
```

Comment:
I saw we have `MathBuilder::constant` and I am under the impression we should use that function (by building a `createMath` builder) rather than  `emitConstantOp`. Can you update the code pls?

Reply:
Yes, I recommend the using of the builder, that also isolate us a bit from constant MLIR changes.

Reply:
I replaced the old code with create.math and create.krnl utilities. Lots of changes, particularly for this file. Not recommend such action for old code unless absolutely needed.